verticapy/tests_new/performance/vertica/collection/test_import_simple.py

```diff
@@ -31,6 +32,16 @@ class TestProfileImport:
     Collection of tests for ProfileImport class
     """
 
+    @pytest.fixture
```

Comment:
The default scope is "function". Is it good enough?

Reply:
Hrm... I made this function scope because `tmp_dir` is also function scope. But I suppose this has more work associated with it. I will change it to be class scope.

Reply:
I needed to change the fixture to use `tmp_path_factory` at class scope. But it worked! Now we will copy files fewer times.